nss-day-cohort-39/tabloid-mvc-depech-mode
# Changing User Profile Types

### Changes
-Added an edit button to the Users in the Index view
-Created an Edit form to allow the user to change UserType
-Created a SQL query to throw an error if the user tries to change the last remaining admin to an author, otherwise update an the type
### To Test
1. Ensure Edit button appear on Users in index view.
2. Try and promote an author to an admin. Try and change that admin back to an author.
3. Try and change all admins to author except for one final active admin. Try and change the final active admin to an author. Ensure that an error message displays when you attempt this.
